data-prepper-api/src/main/java/com/amazon/dataprepper/model/event/JacksonEvent.java

```diff
@@ -57,7 +57,7 @@
 
     private final EventMetadata eventMetadata;
 
-    private final JsonNode jsonNode;
+    protected final JsonNode jsonNode;
```

Comment:
Once you make this protected, it is an exposed interface. Even for protected access, I recommend using methods rather than fields.

So keep this private and add `protected JsonNode getJsonNode()`.